src/Application/components/icons/WelcomeCheckIcon.tsx

```diff
@@ -0,0 +1,12 @@
+const WelcomeCheckIcon = () => (
```

Comment:
This is the same check icon as the `webnative-app-template`, but I passed it through the [SVG optimizer](https://jakearchibald.github.io/svgomg/) I always use to reduce the size by ~50% 👍🏼